fuel/downloaders/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""Download modules for built-in datasets.
+
+Download methods accept a single argument, `save_directory`, which tells
+where to save the downloaded files.
+
+"""
+from fuel.downloaders.binarized_mnist import binarized_mnist
+
+__all__ = ('binarized_mnist',)
```

Comment:
Scrutinizer complained when I was using a list (it said the list was mutable so it couldn't guarantee that everything was tested properly).